api/src/users/entities/comment.entity.ts

```diff
@@ -0,0 +1,23 @@
+import { Column, Entity, JoinColumn, ManyToOne, PrimaryGeneratedColumn } from 'typeorm';
+import { IsString } from 'class-validator';
+import { GenericEntity } from 'src/generic/generic.entity';
+import { User } from './user.entity';
+import { Request } from './request.entity';
+
+@Entity()
+export class Comment extends GenericEntity {
+  @PrimaryGeneratedColumn({ type: 'bigint' })
+  id: number;
+
+  @ManyToOne(type => User)
+  @JoinColumn({ name: 'user_id' })
+  requester: User;
+
+  @ManyToOne(type => Request)
+  @JoinColumn({ name: 'request_id' })
+  volunteer: Request;
```

Comment:
This would be a User too.

Reply:
It's actually supposed to be: 

```
Comment has a user.
Comment belongs to a request.
```

Thank you for catching that though, because it was still very wrong.